src/collection/algorithms/dijkstra.js

```diff
@@ -74,7 +74,7 @@ var elesfn = ({
 
       knownDist[ uid ] = smalletsDist;
 
-      if( smalletsDist === Math.Infinite ){
+      if( smalletsDist === undefined ){
```

Comment:
I initially had it changed to `Infinity` but that was causing tests to fail. Going to take another look at this to see why that is.